docs/_posts/2018-11-19-android.md

```diff
@@ -512,6 +512,41 @@ vendorConfiguration:
       aggregationMode: POOL
 ```
 
+{% endtab %} {% tab file-sync Gradle %}
+
+```kotlin
+marathon {
+    marathon {
+    fileSyncConfiguration {
+      pull.add(
+        new com.malinskiy.marathon.android.configuration.FileSyncEntry(
```

Comment:
FQDN should be simplified to just the simple class name or new package `com.malinskiy.marathon.config.vendor.android` should be used that will be effective as of 0.7.0